parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -1805,3 +1805,48 @@ SqlHostVariable SqlHostVariable() :
     )
     { return new SqlHostVariable(name, getPos()); }
 }
+
+SqlTypeNameSpec ByteDataType() :
+{
+    final SqlTypeName sqlTypeName;
+    final Span s = Span.of();
+    int precision = -1;
+}
+{
+    <BYTE> { sqlTypeName = SqlTypeName.BYTE; }
```

Comment:
nit: We can probably drop the sqlTypeName variable altogether and just plug SqlTypeName.BYTE directly into the call to "new SqlBasicTypeNameSpec" below.

Reply:
Yeah that makes more sense, will update